scripts/build_provenance

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+readonly BUILD_SCRIPT=$1
+readonly OUTPUT_PATH=$2
+
+readonly BUILD_COMMAND="./scripts/docker_run ${BUILD_SCRIPT}"
+eval "${BUILD_COMMAND}"
+readonly BINARY_SHA_256_HASH=$(sha256sum "${OUTPUT_PATH}" | cut -d " " -f 1)
+
+readonly COMMIT_HASH=$(git rev-parse --verify HEAD)
+
+mkdir -p "./provenance"
+mkdir -p "./provenance/${BINARY_SHA_256_HASH}"
+
+readonly PROVENANCE=$(cat << EOF >> "./provenance/${BINARY_SHA_256_HASH}/${COMMIT_HASH}.toml"
+repo = "https://github.com/project-oak/oak"
+commit_hash = "$(git rev-parse --verify HEAD)"
```

Comment:
COMMIT_HASH

Reply:
How comes? This isn't a bash variable, but a part of the string that makes up the toml file expected by transparent release. Afaik all the keys in it are lowercase, why wouldn't this be? See https://github.com/project-oak/transparent-release/blob/main/testdata/build.toml

cc @rbehjati

Reply:
Oh nvm, ignore what I wrote, realized I literally have that bash declaration already 😅